cfgov/paying_for_college/models/blocks.py

```diff
@@ -0,0 +1,57 @@
+from wagtail.wagtailcore import blocks
+
+
+class QuizAnswers(blocks.StructBlock):
+    answers = blocks.ListBlock(
+        blocks.StructBlock([
```

Comment:
I like this idea, @Scotchester , but I have had no luck basing QuizAnswers directly on ListBlock, which requires a child_block argument to base all the elements on. I can't find examples of anyone extending ListBlock in this way.

I hid the stuttered "Answers" label in the admin, but it's a less than ideal solution, because an annoying bare colon still displays. Wagtail's [docs](https://docs.wagtail.io/en/v1.13.4/topics/streamfield.html#basic-block-types) say ListBlock labels default to an empty string, but that doesn't appear to be true.

Reply:
Would this work?

```py
class QuizAnswer(blocks.StructBlock):
    ...

class QuizQuestion(blocks.StructBlock):
    ...
    answers = blocks.ListBlock(
        QuizAnswer(),
        template='paying-for-college/blocks/quiz-answers.html'
    )

class GuidedQuiz(blocks.StructBlock):
   ...
   questions =  blocks.ListBlock(QuizQuestion())
```
[Here's](https://gist.github.com/chosak/1b20fc3a63316d1854433248ad1157b7) the change as a diff. It looks like this in the admin:

<img width="1144" alt="image" src="https://user-images.githubusercontent.com/654645/67516356-85045580-f66e-11e9-89e6-329c605984fd.png">

Reply:
@chosak Thank you, sir. And why didn't I think of that?

Reply:
@chosak Thanks again for lighting the way.

In adjusting the models I learned that it does no good to assign templates to [children of StructBlock and ListBlock](https://docs.wagtail.io/en/v1.13.4/topics/streamfield.html#boundblocks-and-values) -- the templates are ignored. This cleared up much confusion about how templates were working for these models.